### ISSUE TITLE
dev.boringcrypto: suppress git pager pause in misc/boring/release.sh

### ISSUE THREAD

**Issue by katiehockman** [golang/go#46082]
The `misc/boring/release.sh` script requires that the user press Q when git shows the top-level commit in each of the branches. This happens several minutes apart, so requires checking in with the script regularly to make sure that it isn't currently blocked on user-input. We should just suppress these git alerts.

/cc @golang/security

**Comment by seankhliao** [comment]
I think this is no longer relevant due to #51940
